Documentation for CC deployment (#23)
kyguy/strimzi-kafka-operator: documentation/modules/cruise-control/con-cruise-control-overview.adoc

// This concept is included in the following assemblies:
//
// assembly-cruise-control-concepts.adoc

// Save the context of the assembly that is including this one.
// This is necessary for including assemblies in assemblies.
// See also the complementary step on the last line of this file.

:parent-context-cruise-control-overview: {context}

[id='assembly-cruise-control-overview-{context}']

> = Cruise Control

Reviewer: ```suggestion
= Cruise Control overview
```

> Cruise Control is an open source project for simplifying the monitoring and balancing of Kafka partitions across a Kafka cluster.

Reviewer: ```suggestion
Cruise Control is a tool for simplifying the monitoring and rebalancing of partitions across a large-scale Kafka cluster.
```

Reviewer: Kafka partitions > partitions

> Cruise Control is deployed alongside a Kafka cluster to monitor its traffic, propose balanced partition assignments, and trigger partition reassignments based on those proposals.

Reviewer: I suggest using a concise noun phrase, such as "cluster rebalance proposal". I've used this in the next comment.

Please suggest alternatives if that isn't quite right!

Reviewer: I see you're using _partition rebalance proposal_. That works for me.

> Cruise Control is deployed alongside a Kafka cluster to monitor its traffic, propose balanced partition assignments, and trigger partition reassignments based on those proposals.

Reviewer: ```suggestion
You deploy Cruise Control alongside a Kafka cluster to:
* Monitor network traffic
* Propose balanced partition assignments, known as _cluster rebalance proposals_
* Trigger partition reassignments based on those proposals.
```

Reviewer: cpu > CPU

...known as a partition rebalance proposal > known as _partition rebalance proposals_

> 

Reviewer: New line to help with navigation:

The following sections explain how to [deploy Cruise Control](#), [set configuration options](#), [use the Cruise Control API](#), and [execute a cluster rebalance operation](#).

// This concept is included in the following assemblies:
//
// assembly-cruise-control-concepts.adoc

// Save the context of the assembly that is including this one.
// This is necessary for including assemblies in assemblies.
// See also the complementary step on the last line of this file.

:parent-context-cruise-control-overview: {context}

[id='con-cruise-control-overview-{context}']
= Cruise Control overview

> Cruise Control is tool for simplifying the monitoring and rebalancing of partitions across a large-scale Kafka cluster.

Reviewer 2: is *a* tool

Reviewer 2: also it can do it for any-scale cluster, not just "large-scale" ones
